domains.json

```diff
@@ -696,5 +696,7 @@
   "afternic.com",
   "godaddy.co.uk",
   "hostinger.co.uk",
-  "20i.com"
+  "20i.com",
+  "godaddy.in",
+  "hostinger.in"
```

Comment:
This list is taken from ICANN, all domains listed are the official accredited registrars.